snapcraft-options/snapcraft-agent.yaml

```diff
@@ -37,6 +37,7 @@ parts:
     override-build: |
       python3 ./build/build.py --project agent --build-for-snap
       mkdir -p ../install/bin
+      mkdir -p ../install/var/run/deliveryoptimization-agent
```

Comment:
Creating the ../install/var/run/deliveryoptimization-agent is not required I think.

Reply:
Remove the creation of ../install/var/run/deliveryoptimization-agent